release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2732,6 +2732,54 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 
+// 4.15.15
+[id="ocp-4-15-15"]
+=== RHSA-2024:3327 - {product-title} 4.15.15 bug fix and security update
+
+Issued: 2024-05-29
+
+{product-title} release 4.15.15, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:3327[RHSA-2024:3327] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:3332[RHBA-2024:3332] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.15 --pullspecs
+----
+
+[id="ocp-4-15-15-bug-fixes"]
+==== Bug fixes
+
+* Previously, certain situations caused transfer of an Egress IP address from one node to a different node to fail, and this failure impacted the OVN-Kubernetes network. The network failed to send gratuitous Address Resolution Protocol (ARP) requests to peers to inform them of the new node's medium access control (MAC) address. As a reult, peers would temporarily send reply traffic to an old node and this traffic led to failover issues. With this release, the OVN-kubernetes network correctly sends a gratuitous ARP to peers to inform them of the new Egress IP node MAC address, so that each peer can send reply traffic to the new node without causing failover time issues. (link:https://issues.redhat.com/browse/OCPBUGS-33960[*OCPBUGS-33960*]
+
+* Previously, when mirroring operator catalogs, the `oc-mirror` rebuilt the catalogs and regenerated the catalogs internal cache according to the `imagesetconfig` catalog filtering specifications. This operation required the use of the `opm` binary found within the catalogs. In {product-title} 4.15, operator catalogs include the `opm` {op-system-base-full} 9 binary and this caused the mirroring process to fail when running on {op-system-base} 8 systems. With this release, `oc-mirror` no longer builds catalogs by default. Instead, catalogs are mirrored directly to their destination registries. (link:https://issues.redhat.com/browse/OCPBUGS-33575[*OCPBUGS-33575*]
+
+* Previously, the image registry did not support Amazon Web Services (AWS) region `ca-west-1`. With this release, the image registry can now be deployed in this region. (link:https://issues.redhat.com/browse/OCPBUGS-33672[*OCPBUGS-33672*])
+
+* Previously, ServiceAccounts (SAs) could not be used as OAuth2 clients because there were no tokens associated with the SAs. With this release, the OAuth registry client has been modified to anticipate this case and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33210[*OCPBUGS-33210*])
```

Comment:
Either:
```suggestion
* Previously, `ServiceAccount` (SA) objects could not be used as OAuth2 clients because there were no tokens associated with the SAs. With this release, the OAuth registry client has been modified to anticipate this case and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33210[*OCPBUGS-33210*])
```
or:
```suggestion
* Previously, service accounts (SAs) could not be used as OAuth2 clients because there were no tokens associated with the SAs. With this release, the OAuth registry client has been modified to anticipate this case and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33210[*OCPBUGS-33210*])
```